docs/run/retired-systems.mdx

```diff
@@ -71,4 +73,19 @@ jobs = provider.backend.jobs(backend_name=<backend_name>)
 
 #If you want to retrieve a specific job you have the id for
 job = provider.backend.retrieve_job(<job_id>)
+```
+
+### For a job run from Qiskit Runtime
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+# Load your IBM Quantum account(s). Replace "hub/group/project" with your desired instance
+service = QiskitRuntimeService(channel="ibm_quantum", instance="hub/group/project")
+
+# Retrieve a single job by id
+job = service.job("<job_id>")
```

Comment:
Maybe just to keep consistency between the two code snippets, we should remove the quotations around <job_id>, otherwise users might think the two snippets should have differently formatted inputs.  I should have done this in my code example in the original post, my mistake.